src/index.html

```diff
@@ -4,10 +4,265 @@
+      >
+        <section class="main__card card">
+          <div class="card__pictures">
+            <img
+            src="./images/Smart-speaker.png"
+            alt="Bose portable Smart speaker"
+            class="card__img"
+          >
+          </div>
+          <p class="card__item">Bose portable Smart speaker</p>
+          <p class="card__category">Smart home</p>
+          <p class="card__price">$ 399.00</p>
+        </section>
+
+        <section class="main__card card">
+          <div class="card__pictures">
+            <img
+            src="./images/Bluetooth-speaker.png"
+            alt="Bose portable Smart speaker"
+            class="card__img"
+          >
+          </div>
+          <p class="card__item">SoundLink Flex Bluetooth speaker</p>
+          <p class="card__category">Portable bluetooth</p>
+          <p class="card__price">$ 149.00</p>
+        </section>
+
+        <section class="main__card card">
+          <div class="card__pictures">
+            <img
+            src="./images/Color-bluetooth-speaker.png"
+            alt="Bose portable Smart speaker"
+            class="card__img"
+          >
+          </div>
+          <p class="card__item">
+            SoundLink Color Bluetooth speaker II
+          </p>
+          <p class="card__category">Portable bluetooth</p>
+          <p class="card__price">$ 129.00</p>
+        </section>
+      </article>
+      <article class="main__category-box" id="categories">
+        <p class="main__category">Browse Bose <br>products by category</p>
+      </article>
+
+      <article class="main__products products" id="buy">
+        <section
+          class="products__card"
+          data-aos="fade-up"
+          data-aos-duration="1200"
+        >
+          <section class="products__card-pictures">
+            <div class="products__big-img products__big-img--1"></div>
+            <div class="products__small-img products__small-img--1"></div>
+          </section>
+          <p class="products__title">Headphones & earbuds</p>
+        </section>
+
+        <section
+          class="products__card"
+          data-aos="fade-up"
+          data-aos-duration="1200"
+        >
+          <section class="products__card-pictures">
+            <div class="products__big-img products__big-img--2"></div>
+            <div class="products__small-img products__small-img--2"></div>
+          </section>
+          <p class="products__title">Speakers</p>
+        </section>
+
+        <section
+          class="products__card"
+          data-aos="fade-up"
+          data-aos-duration="1200"
+        >
+          <section class="products__card-pictures">
+            <div class="products__small-img products__small-img--3"></div>
+            <div class="products__big-img products__big-img--3"></div>
+          </section>
+          <p class="products__title">Audio sunglasses</p>
+        </section>
+
+        <a href="#" class="products__button">View all</a>
```

Comment:
<img width="387" alt="Screenshot 2023-02-14 at 4 20 24 PM" src="https://user-images.githubusercontent.com/35467335/218765235-15607449-4a9b-46f3-894e-2af8fa8d80a0.png">

You have too big distance between blocks on mobile cause you have bottom padding for your button and top margin for your title.